web/src/components/footer.js

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+import { Link } from 'gatsby';
+import AIRtecAwhite from './svgs/AIRtecAwhite';
+
+export default function Footer() {
+    const externalLinks = [
+        {
+            link: '/',
+            text: 'Link',
+        },
+        {
+            link: '/',
+            text: 'Link',
+        },
+        {
+            link: '/',
+            text: 'Link',
+        },
+    ];
+    const internalLinks = [
+        {
+            link: '/',
+            text: 'Link',
+        },
+        {
+            link: '/',
+            text: 'Link',
+        },
+        {
+            link: '/',
+            text: 'Link',
+        },
+        {
+            link: '/',
+            text: 'Link',
+        },
+        {
+            link: '/',
+            text: 'Link',
+        },
+        {
+            link: '/',
+            text: 'Link',
+        },
+    ];
+    return (
+        <footer className="bg-brand-1 text-ts-body w-full h-96 text-white font-normal fixed bottom-0">
```

Comment:
I think we should set the base font size in global.css. Then we should be able to avoid declaring `text-ts-body` unless we're overwriting existing css.